helm/kube-image-keeper/templates/controller-poddisruptionbudget.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.controllers.pdb.create (gt .Values.controllers.replicas 1.0)  }}
```

Comment:
I don't think we should use the number of replica in this condition. If a user explicitly request the PDB to be created, we should do it, even if there is only 1 replica for now. This could be scaled up later.

It's not true for STS, so it make sense to limit it for registry when in stateful mode, but for a deployment I see no good reason to limit this.